script.js

```diff
@@ -56,6 +61,25 @@ async function searchByName(wantedName) {
     }
 }
 
+async function searchById(queryId) {
+    const response = await fetch("http://localhost:8080/http://api.steampowered.com/ISteamApps/GetAppList/v0002/");
+    const data = await response.json();
+    const matchingApp = data.applist.apps.find(item => item.appid.toString() === queryId)
+    if (matchingApp === undefined) {
+        // There is an error: no apps match this app id
+        displayError("Couldn't find a game with that app id");
+        return;
+    }
+    const gameData = await getGameData(queryId);
+    if (!gameData.success) {
+        // There was an error getting this app from api
+        displayError('Error getting app data')
```

Comment:
When `gameData.success` is `false`, do you know whether there's some key that gives us the actual error message of what went wrong?

If there is, it would be nice to display that (same for the other displayError() on L50).

Reply:
It's just this:

This is the response from:
https://store.steampowered.com/api/appdetails?appids=500400
```json
{
  "500400": {
    "success":false
  }
}
```